docs/serverless-v2/08-01-expose-lambda.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Expose lambdas with API Rules
+type: Tutorials
+---
+
+This tutorial shows how you can expose a lambda function to access it outside the cluster, through an HTTP proxy. To expose it, you use an APIRule custom resource (CR) managed by the in-house API Gateway Controller. This controller reacts to an instance of the APIRule CR and, based on its details, it creates an Istio Virtual Service and Oathkeeper Access Rules that specify your permissions for the exposed function.
+
+This example is based on a simple code which returns the "Hello, World!" string when you access the HTTP host on which the lambda is exposed. This lambda is not secured by an access token (`handler: noop`), and it accepts `GET`, `POST`, `PUT`, and `DELETE` request types.
+
+>**NOTE:** To learn more about securing your lambda, see [this](/components/api-gateway-v2/#tutorials-expose-and-secure-a-service-deploy-expose-and-secure-the-sample-resources) tutorial.
+
+## Steps
+
+Follows these steps:
+
+1. Export these variables:
+
+    ```bash
+    export DOMAIN={DOMAIN_NAME}
+    export NAME={RESOURCE_NAME}
+    export NAMESPACE="kyma-system"
+    ```
+    >**NOTE:** Lambda Service takes the name from the Function CR name. The APIRule CR can have a different name but for the purpose of this tutorial, all related resources share a common name defined under the **NAME** variable.
+
+2. Create a Function CR that specifies the lambda's logic and defines a runtime on which it should run:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: serverless.kyma-project.io/v1alpha1
+    kind: Function
+    metadata:
+      name: $NAME
+      namespace: $NAMESPACE
+    spec:
+      functionContentType: plaintext
+      runtime: nodejs8
+      function: |
+        module.exports = {
+          main: function(event, context) {
+            return 'Hello, World!'
+          }
+        }
+    EOF    
+    ```
+
+3. Check if your lambda was created successfully and has the `Running` status. Run this command:
+
+    ```bash
+    kubectl get functions -n kyma-system
+    ```
+
+4. Create an APIRule CR for your lambda:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: gateway.kyma-project.io/v1alpha1
+    kind: APIRule
+    metadata:
+      name: $NAME
+      namespace: $NAMESPACE
+    spec:
+      gateway: kyma-gateway.kyma-system.svc.cluster.local
+      rules:
+      - path: /.*
+        accessStrategies:
+        - config: {}
+          handler: noop
+        methods:
+        - GET
+        - POST
+        - PUT
+        - DELETE
+      service:
+        host: $NAME.$DOMAIN
+        name: $NAME
+        port: 80
```

Comment:
You can also describe why `80` port. The placeholder Knative Service is exposed on `80` port, so additional information avoid misunderstandings for user.